src/map/Map.h

```diff
@@ -0,0 +1,91 @@
+#ifndef MAP_H
+#define MAP_H
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_image.h>
+#include <SDL2/SDL_ttf.h>
+#include "../game/GameManager.h"
+
+using namespace std;
```

Comment:
using namespace is strictly forbidden in header files, not just by us but everyone.
it will make all files that include it use that namespace regardless of if they want to